### ISSUE TITLE
Can not use both OpenCV 3.1 and JavaCV 1.2 in one app

### ISSUE THREAD

**Issue by viettanbkdn** [bytedeco/javacv#577]
In my app, I compile both OpenCV 3.1 and JavaCV 1.2, and can not start the app. Got error dialog:

> OpenCV error 
> "OpenCV was not initialized correctly. Application will be shut down".

logs:

```
D/OpenCVManager/Helper: Service connection created
D/OpenCVManager/Helper: Trying to get library path
D/OpenCVManager/Helper: Trying to get library list
D/OpenCVManager/Helper: Library list: "libopencv_java3.so"
D/OpenCVManager/Helper: First attempt to load libs
D/OpenCVManager/Helper: Trying to init OpenCV libs
D/OpenCVManager/Helper: Trying to load libs by dependency list
D/OpenCVManager/Helper: Trying to load library /data/app/org.opencv.engine-1/lib/arm64/libopencv_java3.so
D/OpenCVManager/Helper: Cannot load library "/data/app/org.opencv.engine-1/lib/arm64/libopencv_java3.so"
W/System.err: java.lang.UnsatisfiedLinkError: dlopen failed: "/data/app/org.opencv.engine-1/lib/arm64/libopencv_java3.so" is 64-bit instead of 32-bit
W/System.err:     at java.lang.Runtime.load(Runtime.java:332)
W/System.err:     at java.lang.System.load(System.java:1069)
W/System.err:     at org.opencv.android.AsyncServiceHelper.loadLibrary(AsyncServiceHelper.java:347)
W/System.err:     at org.opencv.android.AsyncServiceHelper.initOpenCVLibs(AsyncServiceHelper.java:373)
W/System.err:     at org.opencv.android.AsyncServiceHelper.access$000(AsyncServiceHelper.java:18)
W/System.err:     at org.opencv.android.AsyncServiceHelper$3.onServiceConnected(AsyncServiceHelper.java:300)
W/System.err:     at android.app.LoadedApk$ServiceDispatcher.doConnected(LoadedApk.java:1335)
W/System.err:     at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:1352)
W/System.err:     at android.os.Handler.handleCallback(Handler.java:739)
W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:95)
W/System.err:     at android.os.Looper.loop(Looper.java:158)
W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:7224)
W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
W/System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
D/OpenCVManager/Helper: First attempt to load libs fails
D/OpenCVManager/Helper: Init finished with status 255
D/OpenCVManager/Helper: Unbind from service
D/OpenCVManager/Helper: Calling using callback
E/OpenCVLoader/BaseLoaderCallback: OpenCV loading failed!
```

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacpp-presets/issues/52#issuecomment-100716828

**Comment by saudet** [comment]
64-bit builds for Android (as the "android-arm64" and "android-x86_64" platforms) are now available, so please give it a try! Binaries for 1.4.1-SNAPSHOT can be downloaded from the snapshot repository: http://bytedeco.org/builds/
